postgres/docker-healthcheck

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eo pipefail
 
-host="$(hostname --ip-address || echo '127.0.0.1')"
+host="$(hostname -i || echo '127.0.0.1')"
```

Comment:
`hostname(1)` in Alpine only accepts short-form flags (since it comes from BusyBox)